pkg/util/resourcegrouptag/resource_group_tag.go

```diff
@@ -125,6 +125,12 @@ func GetFirstKeyFromRequest(req *tikvrpc.Request) (firstKey []byte) {
 				}
 			}
 		}
+	case *kvrpcpb.PessimisticLockRequest:
```

Comment:
Why add this new case?

Reply:
I just think it was missing, so I added it.